python/tk_desktop/project_commands_model.py

```diff
@@ -186,22 +187,41 @@ def add_command(self, name, button_name, menu_name, icon, command_tooltip, group
                 menu_item.setData(button_name, self.BUTTON_NAME_ROLE)
                 menu_item.setData(menu_name, self.MENU_NAME_ROLE)
                 menu_item.setData(name, self.COMMAND_ROLE)
+                menu_item.setData(is_menu_default, self.IS_MENU_DEFAULT_ROLE)
                 menu_item.setToolTip(command_tooltip)
                 if icon is not None:
                     menu_item.setIcon(icon)
                 item.appendRow(menu_item)
 
     @classmethod
     def get_item_children_in_order(cls, item):
+        """
+        Sort the item's children in reverse 'version' order based on the menu name
+        of the item. Item children whose IS_MENU_DEFAULT_ROLE data value is True
+        will be prepended to the list.
+
+        :param QtGui.QStandardItem item: Input project button to sort child menu items for.
+        :returns: List of sorted menu items
+        """
+        # Theoretically, there should only be one "default" child in the list of item
+        # children, but since this is a human configurable value, there may 'accidentally'
+        # be more. Rather than raising an error, attempt to handle this case gracefully
+        # by keeping track of default and non-default children separately. Sort each
+        # list in reverse version number order, then add them together to construct
+        # the list of ordered item children to return.
```

Comment:
an example here would help a great deal. also, why not move the comment up in the doc string?